test/test_gpu_tensorcore.cpp

```diff
@@ -4211,6 +4211,51 @@ TEST_F(NVFuserTest, FusionAmpereMatmulSmemEpilogueRelu_CUDA) {
     }
   }
 }
+
+// Test the matmul scheduler's single-kernel split-K support
+TEST_F(NVFuserTest, FusionTuringMatmulSplitK_CUDA) {
```

Comment:
According to https://github.com/NVIDIA/Fuser/pull/1002/files#r1347852745, could you also add a test for split-k of batched matmul?